### PR TITLE
Ingest Gobblin tracking events into wherehows by Kafka consumer client

### PR DESCRIPTION
**Pull request by alyiwang**
Also port generic Kafka schemaRegistry and serializers into wherehows-common.

### CODE REVIEW

File: backend-service/app/utils/KafkaConfig.java [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    }
+  }
+
+  /**
+   * get Kafka properties from database, also update _props
+   * @return Properties
+   * @throws Exception
+   */
+  public static Properties getKafkaProperties() throws Exception {
+    final List<Map<String, Object>> rows = JdbcUtil.wherehowsJdbcTemplate.queryForList(GET_KAFKA_PROPERTIES);
```
Comment (ericsun2) [inline backend-service/app/utils/KafkaConfig.java]: Let's use wh_etl_job_property to store the Kafka configuration

Multi-value properties can be expressed as comma-separated list.

**Comment by ericsun2** [comment]
LGTM
Go Kafka!

**Comment by planckiii** [comment]
There will be some documentation on wiki related to that component ?

**Comment by SunZhaonan** [comment]
Congrats on this new milestone! It would be great to see some general docs under wherehows/wiki @alyiwang

**Pull request merged**